test/JDBC/expected/BABEL-2844.out

```diff
@@ -21,7 +21,7 @@ Query Text: create table babel_2844_t1 (a1 int, b1 int);
 
 ~~START~~
 text
-Babelfish T-SQL Batch Parsing Time: 64.545 ms
+Babelfish T-SQL Batch Parsing Time: 64.592 ms
```

Comment:
these time values cannot be in expected results -- they will always be different

Reply:
Yes, our JDBC framework is capable to ignore these diff.